.gitignore

```diff
@@ -15,3 +15,11 @@ cypress/videos
 cypress/fixtures
 cypress/plugins
 .DS_Store
+
+# Recommendation from https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored (not using Zero-installs)
+.yarn/*
+!.yarn/releases
+!.yarn/plugins
+!.yarn/sdks
+!.yarn/versions
+.pnp.*
```

Comment:
@bryanph as far as i can tell, the project is using zero-installs, but .gitignore is setup for non-zero installs. In case PnP was the preferred implementation, here's the updated .gitignore.

```suggestion
# Recommendation from https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored (using Zero-installs)
.yarn/*
!.yarn/cache
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```

Reply:
what do you mean we use zero-installs right now? that's a yarn berry feature :thinking:. We don't commit dependencies right now so I assumed that's what we wanted to continue doing.

Reply:
Fair point, I've been thinking about pnp.